test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -473,25 +482,125 @@ public void onCommit(Map<String, List<CorfuStreamEntry>> mutations) {
             }
         }
 
-        try (TxnContextShim txn = shimStore.txn(someNamespace)) {
-            txn.delete(tableName, key);
-            txn.commit((mutations) -> {
-                 mutations.values().forEach(mutation -> {
-                     CorfuStreamEntry.OperationType op = mutation.get(0).getOperation();
-                     assertThat(op).isEqualTo(CorfuStreamEntry.OperationType.DELETE);
-                 });
+        CommitCallbackImpl commitCallback = new CommitCallbackImpl();
```

Comment:
Where is this commitCallback used?

Reply:
it is used by verticals to extract the modified metadata. It can also serve as a way to do fast table notifications.

Reply:
But I couldn't find the usage of this `commitCallback` variable in this test. The following `txn.addCommitCallback()` adds its own callback lambda function right?